src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -9036,5 +9036,46 @@ void M()
     }
 }", "C");
         }
+
+        [WorkItem(14163, "https://github.com/dotnet/roslyn/issues/14163")]
+        [Fact]
+        [Trait(Traits.Feature, Traits.Features.Completion)]
+        [Test.Utilities.CompilerTrait(Test.Utilities.CompilerFeature.LocalFunctions)]
+        public async Task LocalFunctionDescription()
+        {
+            await VerifyItemExistsAsync(@"
+class C
+{
+    void M()
+    {
+        void Local() { }
+        
+        $$
+    }
+}", "Local", "void Local()");
```

Comment:
FYI, we probably want to say something like "(local function) void Local()".  But that can be considered a feature request if you want to file a bug on it.